pom.xml

```diff
@@ -116,6 +116,11 @@
       <version>1.3</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
is it for test? If so, it should be "test“ scoped

Reply:
Probably only for test, I will add this. If I don't add this, I get warnings from the logger. I'm not sur If the logger is only for dev, but I will assume that.